src/components/DashBoard/TableOrders/TableAllOrders.jsx

```diff
@@ -4,17 +4,22 @@ import "./table.css";
 import { Icon } from "@iconify/react";
 import ModalPortal from "../../modal";
 import useUser from "../../hooks/UseUser";
+import img from "./img/favicon-32x32.png"
+
 
 const OrdersTableAll = () => {
     const { user } = useUser();
     const [ordersData, setOrdersData] = useState([]);
     const [modalData, setModalData] = useState(null);
     const [showModal, setShowModal] = useState(false);
+    const [modalDataPas, setModalDataPas] = useState(null);
+    const [showModalPas, setShowModalPas] = useState(false);
     const [currentPage, setCurrentPage] = useState(1);
     const [cuantityPage, setCuantityPage] = useState(0);
+    const [pasInfo, setPasInfo] = useState();
     const resultsPerPage = 7;
     const userId =
-        user.type === "superadmin" ? "" : user.type === "admin" ? "" : user.id;
+        user.type === "superadmin" ? "" : user.type === "admin" ? "" : user.id_user;
```

Comment:
```suggestion
        user.type !== "superadmin" && user.type !== "admin" ? user.id_user : null;
```
i think you can reduce this ternary like this.

Reply:
This ternary return null and need ""

Reply:
then return "", its ok